### PR TITLE
PLU-318: [TEMPLATES-9] Add UI changes based on stacey review

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem
Users are unaware of what they can do with Plumber.

## Solution
Introduce templates to speed up their pipe creation and provide new ideas for them to automate. 

## Details
- Edited `helpMessage` for the different apps and events
- Update templates data
- Modify infobox and add some CSS changes
- Use `skeleton` instead of `spinner` for template tile

## Tests
- [x] Templates still work
- [x] Tile templates show the different infobox message for create row, find single row and update single row

### CODE REVIEW

**Comment by m0nggh** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/725?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#725** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/725?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#748** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/748?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 13 other dependent PRs ([#714](https://github.com/opengovsg/plumber/pull/714) <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/714?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#715](https://github.com/opengovsg/plumber/pull/715) <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/715?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#716](https://github.com/opengovsg/plumber/pull/716) <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/716?utm_source=stack-comment-icon" target="_blank">Graphite</a> and 10 others)
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @m0nggh and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/src/pages/Templates/index.tsx [hunk 1]
```diff
@@ -62,15 +64,17 @@ export default function Templates(): JSX.Element {
...
-                <Flex flexDir="column" gap={2} mt={2}>
-                  <Text textStyle="subhead-1">{template.name}</Text>
-                  <Text textStyle="body-2">{template.description}</Text>
-                </Flex>
+                <Skeleton isLoaded={!loading}>
+                  <Flex flexDir="column" gap={2} mt={2}>
+                    <Text textStyle="subhead-1">{template.name}</Text>
+                    <Text textStyle="body-2">{template.description}</Text>
+                  </Flex>
+                </Skeleton>
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Templates/index.tsx]: this skeleton will never appear

File: packages/frontend/src/pages/Templates/index.tsx [hunk 2]
```diff
@@ -42,16 +41,19 @@ export default function Templates(): JSX.Element {
...
-        {loading ? (
-          <Center my={12}>
-            <PrimarySpinner fontSize="4xl" />
+        {/* Sanity check if our templates file is missing */}
+        {!templates || templates.length === 0 ? (
+          <Center my={24}>
+            <Text>
+              There are no templates now, please contact support@plumber.gov.sg
+            </Text>
           </Center>
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Templates/index.tsx]: !templates could also mean it's still loading

**Review by pregnantboy — CHANGES_REQUESTED** [review]
image
 can horizontally align the button and title to the border

File: packages/frontend/src/pages/Templates/index.tsx (hunk 2 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Templates/index.tsx]: actually i dont need this already

File: packages/frontend/src/pages/Templates/index.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Templates/index.tsx]: huh if i click to templates page i see it though

File: packages/frontend/src/pages/Templates/index.tsx [hunk 3]
```diff
@@ -38,39 +37,35 @@ export default function Templates(): JSX.Element {
...
+            <Tile
+              key={index}
+              icon={() => (
+                <Box bg="primary.100" p={2} borderRadius="base">
+                  <TemplateIcon iconName={template.iconName} />
+                </Box>
+              )}
+              onClick={() => navigate(URLS.TEMPLATE(template.id))}
+            >
+              <Skeleton isLoaded={!loading}>
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Templates/index.tsx]: the skeleton will never show if templates is undefined. can just remove

**Review by pregnantboy — APPROVED** [review]
nit: remove skeleton

File: packages/frontend/src/pages/Templates/index.tsx (hunk 3 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Templates/index.tsx]: ok i get you but q weird the skeleton still shows by default or smth?

**Pull request merged**